docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,189 @@
+    ```
+
+- To install a pre-release or development version:
+
+    ```
+    python -m pip install mbed-tools --pre
+    ```
+
+## Upgrade
+
+Use pip to upgrade your version:
+
+```
+python -m pip install mbed-tools --upgrade
+```
+
+# Use
+
+Currently, the new Mbed Tools supports three boards: `K64F`, `DISCO_L475VG_IOT01A` and `NRF52840_DK`.
+
+For help:
+
+- To get help for all commands, use:
+
+    ```
+    mbed-tools --help
+    ```
+
+- To get help for a specific command, use `mbed-tools <command> --help`. For example, for helping with listing connected devices (the `devices` command):
+
+    ```
+    mbed-tools devices --help
+    ```
+
+## Create a project
+
+You can create a new project or create a local copy of one of our example applications.
+
+### Create a new project
+
+To create a new Mbed OS project in a specified path:
+
+- To create the project and download a new copy of Mbed OS (latest official release):
+
+    ```
+    mbed-tools init <PATH>
+    ```
+
+    The path can be:
+    - Absolute. `init` will create the folder if it doesn't exist.
+    - Relative.
+
+    If you have already created a project folder, you can use `.`
+
+    If you want the `init` command to create a project folder, use `.\<folder-name>`.
+
+### Use an example application
+
+To create a local copy of an example application, use the `clone` command with the example name listed below:
+
+```
+mbed-tools clone <example> <PATH>
+````
+
+- [mbed-os-example-blinky](https://github.com/ARMmbed/mbed-os-example-blinky)
+- [mbed-os-example-ble](https://github.com/ARMmbed/mbed-os-example-ble) - use the BLE button example.
+- [mbed-os-example-cellular](https://github.com/ARMmbed/mbed-os-example-cellular)
+- [mbed-os-example-devicekey](https://github.com/ARMmbed/mbed-os-example-devicekey)
+- [mbed-os-example-kvstore](https://github.com/ARMmbed/mbed-os-example-kvstore)
+- [mbed-os-example-lorawan](https://github.com/ARMmbed/mbed-os-example-lorawan)
+- [mbed-os-example-mbed-crypto](https://github.com/ARMmbed/mbed-os-example-mbed-crypto)
+- [mbed-os-example-nfc](https://github.com/ARMmbed/mbed-os-example-nfc)
+- [mbed-os-example-sockets](https://github.com/ARMmbed/mbed-os-example-sockets)
+
+
+## Configure the project
+
+
+### Project environment variables
+
+Mbed Tools has two environment variables that you can set for a project:
+
+- `MBED_API_AUTH_TOKEN`: Token to access private board information stored for a vendor team.
+- `MBED_DATABASE_MODE`: Use online or offline mode. Possible values:
+    - `AUTO`: Search the offline database first; search the online database only if the board wasn't found offline. This is the default value.
+    - `ONLINE`: Alway use the online database.
+    - `OFFLINE`: Always use the offline database.
+
+To set values, create an `.env` file in the root directory of the project. The file should contain definitions in the `<VARIABLE>=<value>` format.
+
+For more information, use `mbed-tools env --help`
```

Comment:
`mbed-tools env`

Reply:
Have we figured out what does what? I found the Slack conversation really confusing